doc/guide/bindings.hpp

```diff
@@ -1214,9 +1285,8 @@ binding.
 
 @subsection bindings_python_testing Testing the Python bindings
 
-We cannot do our tests only from the Boost Unit Test Framework in C++ because we
-need to see that we are able to load parameters properly from Python and return
-output correctly.
+We cannot do our tests only from the Catch in C++ because we need to see that we
+are able to load parameters properly from Python and return output correctly.
```

Comment:
I think this is difficult to read, maybe just remove this section and in the next section, mention that besides the unit tests written with the Catch framework we also have auxiliary Python tests to test the specific binding functionality?

Reply:
Sorry it took me so long to get back to this.  I tried updating this bit in 5fb3ed04e9; what do you think?